src/Components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,60 @@
+import "./sidebar.css";
+import {
+  MdExplore,
+  MdOutlineWatchLater,
+  CgPlayList,
+  RiHistoryLine,
+  BiLike,
+} from "../../Utils/icons";
+import { Link, useLocation } from "react-router-dom";
+export const Sidebar = () => {
+  const { pathname } = useLocation();
+  return (
+    <div>
+      <aside className="sidebar">
+        <div className="pages">
+          <Link
+            className={`category-names ${
+              pathname === "/" ? "active-tab" : "white"
+            } `}
```

Comment:
There's a better way to style the `Active` link.
Check this out https://v5.reactrouter.com/web/api/NavLink